enterprise/views.py

```diff
@@ -833,7 +861,15 @@ def post(self, request):
         defer_creation = request.POST.get('defer_creation')
         consent_provided = bool(request.POST.get('data_sharing_consent', False))
         if defer_creation is None and consent_record.consent_required():
-            if course_id:
+
+            # Skip creating EnterpriseCourseEnrollment record if we found course_key instead of course_run_id
```

Comment:
this comment is sounded a little confusing at first. can we change starting like that?
```
Creating EnterpriseCourseEnrollment record if we found course_run_id instead of course_key in the course_id param
```
something like that.